info.yml

```diff
@@ -18,6 +18,30 @@ configuration:
                      commands that do not match any of the values in the groups
                      setting."
 
+    run_at_startup:
+        type: list
+        description: "Controls what apps will run on startup.  This is a list
+                     where each element is a dictionary with two keys:
+                     'app_instance' and 'name'.
+                     The 'app_instance' value connects this entry to a
+                     particular app instance defined in the environment
+                     configuration file.
+                     The 'name' is the menu name of the command to run when
+                     desktop starts up. If 'name' is '', then all commands from
+                     the given app instance are started.
+
+                     To add the 'Apps' tab (which contains the app launcher),
+                     add the following startup command to the list:
```

Comment:
@robblau Oops, you are right! Forgot about `get_setting`'s default value and thought you meant `info.yml`'s `default_value`. Updated! (not sure how to quickly test the `add_engine` use-case though - does the code look right to you?)